consensus/src/chained_bft/block_storage/block_store_test.rs

```diff
@@ -397,48 +397,6 @@ fn test_need_fetch_for_qc() {
     );
 }
 
-#[test]
```

Comment:
unclear why this test was removed

Reply:
the function now becomes very simple, and craft a LedgerInfo for the case needs additional helper function which I feel not worth the effort, but let me know your thought